runtime/lua/vim/_editor.lua

```diff
@@ -316,18 +316,29 @@ do
   end
 end
 
---- Defers callback `cb` until the Nvim API is safe to call.
+--- Returns a function which calls {callback} via |vim.schedule()|.
+---
+--- The returned function passes all arguments to {callback}.
+---
+--- Example:
+---
+--- ```lua
+--- function notify_readable(_err, readable)
+---   vim.notify("readable? " .. tostring(readable))
+--- end
+--- vim.uv.fs_access(vim.fn.stdpath("config"), "R", vim.schedule_wrap(notify_readable))
+--- ```
 ---
 ---@see |lua-loop-callbacks|
 ---@see |vim.schedule()|
 ---@see |vim.in_fast_event()|
----@param cb function
+---@param callback function
 ---@return function
-function vim.schedule_wrap(cb)
+function vim.schedule_wrap(callback)
```

Comment:
Going to insist on "fn" since I don't want to see "callback" proliferate. Functions are just functions, usually.

```suggestion
function vim.schedule_wrap(fn)
```

Reply:
100%. Can I also change `vim.schedule`?